Unresolved import `nanoid::nanoid`?

Hi, rust newbie here. I just created a new project to try out `nanoid`, but it says `error[E0432]: unresolved import 'nanoid::nanoid'`.

`Cargo.toml`:
```toml
[package]
name = "nanoid_test"
version = "0.1.0"
authors = ["Exr0n <spotyie@gmail.com>"]
edition = "2018"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
nanoid = "0.1.3"
```

`src/main.rs`:
```rust
extern crate nanoid;
use nanoid::nanoid;

fn main() {
    println!("Hello, world!");
}

```

`$ cargo run --verbose`:
```
       Fresh libc v0.2.69
       Fresh rand v0.4.6
       Fresh nanoid v0.1.3
   Compiling nanoid_test v0.1.0 (/Users/exr0n/Desktop/snap/nanoid_test)
     Running `rustc --crate-name nanoid_test --edition=2018 src/main.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type bin --emit=dep-info,link -C debuginfo=2 -C metadata=e6ce498e3c0aac20 -C extra-filename=-e6ce498e3c0aac20 --out-dir /Users/exr0n/Desktop/snap/nanoid_test/target/debug/deps -C incremental=/Users/exr0n/Desktop/snap/nanoid_test/target/debug/incremental -L dependency=/Users/exr0n/Desktop/snap/nanoid_test/target/debug/deps --extern nanoid=/Users/exr0n/Desktop/snap/nanoid_test/target/debug/deps/libnanoid-8c29b7d0393c4491.rlib`
error[E0432]: unresolved import `nanoid::nanoid`
 --> src/main.rs:2:5
  |
2 | use nanoid::nanoid;
  |     ^^^^^^^^^^^^^^ no `nanoid` in the root

error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.
error: could not compile `nanoid_test`.

Caused by:
  process didn't exit successfully: `rustc --crate-name nanoid_test --edition=2018 src/main.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type bin --emit=dep-info,link -C debuginfo=2 -C metadata=e6ce498e3c0aac20 -C extra-filename=-e6ce498e3c0aac20 --out-dir /Users/exr0n/Desktop/snap/nanoid_test/target/debug/deps -C incremental=/Users/exr0n/Desktop/snap/nanoid_test/target/debug/incremental -L dependency=/Users/exr0n/Desktop/snap/nanoid_test/target/debug/deps --extern nanoid=/Users/exr0n/Desktop/snap/nanoid_test/target/debug/deps/libnanoid-8c29b7d0393c4491.rlib` (exit code: 1)
```

It seems like `rustc` knows where to find the `nanoid` crate, but for some reason can't use it while compiling? I can use other crates like `chrono` fine, so I have no clue what's going on.

--- a/README.md
+++ b/README.md
@@ -25,6 +25,13 @@ and has a similar number of unique IDs in just 21 symbols instead of 36.
 
 ## Usage
 
+### Install
+
+```toml
+[dependencies]
+nanoid = "0.3.0"
+```
+
 ### Simple
 
 The main module uses URL-friendly symbols (`A-Za-z0-9_-`) and returns an ID

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -8,7 +8,7 @@
 //!
 //! ```toml
 //! [dependencies]
-//! nanoid = "0.1.3"
+//! nanoid = "0.3.0"
 //! ```
 //!
 //! ```rust

